main/acle.md

```diff
@@ -5064,3 +5071,91 @@ transaction.
 | uint64_t __ttest (void)                       | -                | Xt -> result   | ttest <Xt>        |
 
 These intrinsics are available when `arm_acle.h` is included.
+
+# Architectural Extension Bridges
+
+## NEON-SVE Bridge
+
+The NEON_SVE Bridge adds intrinsics that allow conversions between NEON and
+SVE vectors.
```

Comment:
Nit:
```suggestion
The NEON_SVE Bridge adds intrinsics that allow conversions between NEON and SVE vectors.
```

Reply:
do we not care about the 80 char line limit anymore?

Reply:
Yes, we do. Ignore my comment. I got confused by a line wrap while reviewing this on a small screen. Apologies for the distraction.